influxdata/giraffe
# chore: move builds to webpack

Closes #324 

## Testing process
This is a bit involved. It makes use of the fact that [yarn can pull commits from github.](https://stackoverflow.com/a/43636577/92446)

1. from `giraffe/giraffe`, work in a branch off master
2. build a tarball representing this release
  ```sh
  yarn build
  npm pack
  ```
3. get the tarball out on the internet where it's accessible
  ```sh
  git add .
  git commit -m "npm dist tarball"
  git push origin $branch_name
  ```
4. go to your branch on github and find the tarball you just pushed up. Copy a link to its **raw** url.
5. from `ui`,
  ```sh
  yarn add $copied_link_to_raw_tarball_url
  ```
6. from 'monitor-ci'
  ```sh
  make build NODE=chronograf
  make start NODE=chronograf
  make logs NODE=chronograf
  ```
7. from there, you should be able to test the changes in your local ui
